libraries/botbuilder-dialogs/src/prompts/oauthPrompt.ts

```diff
@@ -229,9 +228,9 @@ export class OAuthPrompt extends Dialog {
         const isMessage: boolean = dc.context.activity.type === ActivityTypes.Message;
         const isTimeoutActivityType: boolean =
             isMessage ||
-            this.isTokenResponseEvent(dc.context) ||
-            this.isTeamsVerificationInvoke(dc.context) ||
-            this.isTokenExchangeRequestInvoke(dc.context);
+            OAuthPrompt.isTokenResponseEvent(dc.context) ||
```

Comment:
Since you're here could you switch these to `??`

Reply:
They are boolean values, so I believe we should use `||`.

Reply:
Ah - of course, oversight on my part. Thanks!